shared_fields_common.view.lkml

```diff
@@ -114,6 +114,15 @@ view: shared_fields_common {
     group_label: "Location"
   }
 
+  dimension: city_and_region {
+    type: string
+    sql: CASE
+       WHEN (${TABLE}.geo_country = 'CA') THEN ${TABLE}.geo_city || ', ' || ${TABLE}.geo_region_name
```

Comment:
sometimes there is no value in `geo_city`, so there are a lot of "(blank), COUNTY" type results when filtering on the field.